app/components/admin/roles/table_actions_component.rb

```diff
@@ -18,7 +18,8 @@ def already_has_role?
     end
 
     def add_user_text
-      t("admin.#{role}.#{role.singularize}.add")
+      role_key = role.tr("/", "_")
```

Comment:
Instead of doing this, how about changing the YAML files to use `sdg/manager` (I think we usually do this with ActiveRecord classes :thinking:)?